source/common/tracing/zipkin/span_context.cc

```diff
@@ -0,0 +1,122 @@
+#include "common/common/utility.h"
+
+#include "common/tracing/zipkin/zipkin_core_constants.h"
+#include "common/tracing/zipkin/span_context.h"
+
+namespace Zipkin {
+
+// String that separates the span-context fields in its string-serialized form.
+const std::string SpanContext::FIELD_SEPARATOR_ = ";";
+
+// String value corresponding to an empty span context.
+const std::string SpanContext::UNITIALIZED_SPAN_CONTEXT_ = "0000000000000000" + FIELD_SEPARATOR_ +
```

Comment:
See other comments on constructor/destructor ordering across modules. Strongly prefer to allocate non-POD static/globals on heap via function local static pointers and leave them there.